### PR TITLE
DIGI 15 - Create Tax Page

### PR DESCRIPTION
**Pull request by anebhargavkrishna**
1.Added End point
2.Added LibraryUtility
3.Added TaxpageLibrary
4.Added TaxPageController.cs
5.And view to taxpagecontroller

### CODE REVIEW

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs [hunk 1]
```diff
@@ -51,30 +55,63 @@ public async Task<HttpResponseMessage> GetAPISync(string endPoint)
         /// <returns></returns>
         public async Task<HttpResponseMessage> POSTAsync(string endPoint, string Json)
         {
-
-            string apiUrl = Convert.ToString(ConfigurationManager.AppSettings["digiapiurl"]) + endPoint;
-
+            Task<HttpResponseMessage> response = null;
+            var val = getTokenDetails();
```
Comment (RAMUNIMMALA) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs:59]: should be with in try block and token verification is required

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/TaxPageLibrary.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+using Newtonsoft.Json.Linq;
+
+
+namespace DigiOrderEnterpriseLibrary
+{
+    public class TaxPageLibrary
+    {
+        LibraryUtility libraryUtility = null;
+        Task<HttpResponseMessage> httpResponse = null;
+        public Task<HttpResponseMessage> CreateTaxPage(string Title, string Description)
```
Comment (RAMUNIMMALA) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/TaxPageLibrary.cs:13]: provide line gap between variables and methods

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxPageController.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            {
+                taxPageLibrary = new TaxPageLibrary();
+                taxPageLibrary.CreateTaxPage(TaxModel.TITLE, TaxModel.DESCRIPTION);
+            }
+            catch (Exception ex)
+            {
+                TempData["ErrorMessage"] = "Failed to insert record";
+                TempData.Keep();
+            }
+            ViewBag.ControlNviation = "Taxpage";
```
Comment (RAMUNIMMALA) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxPageController.cs:31]: Correct the spelling

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/TaxPageLibrary.cs (hunk 2 above)
Comment (anebhargavkrishna) [reply DigiOrderEnterprise/DigiOrderEnterpriseLibrary/TaxPageLibrary.cs:13]: provided line gap

Comment (anebhargavkrishna) [reply DigiOrderEnterprise/DigiOrderEnterpriseLibrary/TaxPageLibrary.cs:13]: Provided line gap

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxPageController.cs (hunk 3 above)
Comment (anebhargavkrishna) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/TaxPageController.cs:31]: Changed as per comment

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs (hunk 1 above)
Comment (anebhargavkrishna) [reply DigiOrderEnterprise/DigiOrderEnterpriseLibrary/LibraryUtility.cs:59]: Added Variable in try block

**Pull request merged**